### DISCUSSION TITLE
Easy add + generate: bundle init rails new blog for fast bootstrapping ✨

### DISCUSSION THREAD

**Discussion by emilebosch** [ruby/rubygems#6652] [Ideas]
So just a quick interesting idea. Oftentimes you want to bootstrap a project quickly. It would be nice if that was possible in such a way, that you both can create the bundler file, add the gem, and run a command with its primary executable of said gem.
A super fast way to init a project and start, basically simplifies a couple of commands:

```sh
bundle init rails new blog
```

Becomes

```sh
bundle init
bundle add rails
bundle exec rails <whatever the user types>
```

Would work for the projects where the name is the same as the primary executable.

**Comment by simi** [comment]
Hello! Feel free to check recently added `gem exec` command. Ensure you're running latest rubygems (`gem update --system`) and type `gem exec --help`. You can find more info at https://github.com/rubygems/rfcs/blob/master/text/0008-gem-exec.md.

**Comment by emilebosch** [comment]
Hi @simi! So fun hahah!  i've proposed the gem exec back in 2019 in https://github.com/rubygems/rubygems/issues/2872. The reason i never continued is because i didnt feel 'right' .

**Comment by deivid-rodriguez** [comment]
Closing since `gem exec` is now a thing.
